openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -315,6 +315,21 @@ def get_stats_for_single_hypervisor(self, hyp, custom_tags=None,
                 metric_label = "openstack.nova.{}".format(label)
                 self.gauge(metric_label, val, tags=tags)
 
+        # This makes a request per hypervisor and only sends hypervisor_load 1/5/15
+        # Disable this by default for higher performance in a large environment
+        # If the Agent is installed on the hypervisors, system.load.1/5/15 is available
```

Comment:
```suggestion
        # If the Agent is installed on the hypervisors, system.load.1/5/15 is available as a system metric
```

Just to help reinforce that its not coming from this check.